Lib/_strptime.py

```diff
@@ -586,3 +591,32 @@ def _strptime_datetime(cls, data_string, format="%a %b %d %H:%M:%S %Y"):
         args += (tz,)
 
     return cls(*args)
+
+def _strptime_datetime_date(data_string, format):
+    """Return a date based on the input string and the format string."""
+    if not format:
+        raise ValueError("Date format is not valid.")
```

Comment:
This line and its equivalent in `_time` are not being hit. If I understand correctly this branch is only hit if `format` is empty?

Reply:
useless, thanks